android/app/build.gradle

```diff
@@ -25,6 +25,14 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
+project.ext.envConfigFiles = [
+        stagingdebug: ".env.staging",
+        stagingrelease: ".env.staging",
+        productiondebug: ".env",
+        productionrelease: ".env"
+]
+apply from: project(':flutter_config').projectDir.getPath() + "/dotenv.gradle"
```

Comment:
Have you tested flutter_config applying on ios yet? It requires applying some more scripts here https://github.com/ByneappLLC/flutter_config/blob/master/doc/IOS.md 💭 

Please add POW for iOS 🙏

Reply:
@chornerman I think you missed this issue 👋